sublime_linter.py

```diff
@@ -363,124 +379,106 @@ def display_errors(self, view, tooltip=False):
+        msg_dict = persist.errors.get(view.id())
+        if not msg_dict:
+            return
 
         # Get the line number of the first line of the first selection.
         try:
-            lineno = view.rowcol(view.sel()[0].begin())[0]
+            lineno, colno = view.rowcol(view.sel()[0].begin())
         except IndexError:
-            lineno = -1
-
-        if vid in persist.errors:
-            errors = persist.errors[vid]
-
-            if errors:
-                lines = sorted(list(errors))
-                counts = [len(errors[line]) for line in lines]
-                count = sum(counts)
-                plural = 's' if count > 1 else ''
-
-                if lineno in errors:
-                    # Sort the errors by column
-                    line_errors = sorted(errors[lineno], key=lambda error: error[0])
-                    line_errors = [error[1] for error in line_errors]
-
-                    if plural:
-                        # Sum the errors before the first error on this line
-                        index = lines.index(lineno)
-                        first = sum(counts[0:index]) + 1
-
-                        if len(line_errors) > 1:
-                            last = first + len(line_errors) - 1
-                            status = '{}-{} of {} errors: '.format(first, last, count)
-                        else:
-                            status = '{} of {} errors: '.format(first, count)
-                    else:
-                        status = 'Error: '
-
-                    status += '; '.join(line_errors)
-
-                    if persist.settings.get('tooltips') and tooltip:
-                        self.open_tooltip(lineno, line_errors)
-                else:
-                    status = '%i error%s' % (count, plural)
+            lineno, colno = -1, -1
 
-                view.set_status('sublimelinter', status)
-            else:
-                view.erase_status('sublimelinter')
+        line_dict = msg_dict.get(lineno)
+        if not line_dict:
+            return
 
-    def get_active_view(self):
-        """Return the active view in the currently active window."""
+        msgs = []
+        point = view.text_point(lineno, colno)
 
-        return sublime.active_window().active_view()
+        for error_type, dc in line_dict.items():
+            for d in dc:
+                region = d.get("region")
+                if region:
+                    if region.contains(point):
+                        msgs.append(d["msg"])
+                elif colno == 0:
+                    msgs.append(d["msg"])
 
-    def get_template(self):
-        """
-        If no tooltip theme has been configured, returns False.
 
-        Otherwise, returns a Template object with the template text.
+        def count_msgs(key):
+            return len(line_dict.get(key, []))
 
-        """
-        tooltip_theme = persist.settings.get('tooltip_theme')
+        status = "SublimeLinter: ⚠: {} ⮾: {}".format(
```

Comment:
These unicode characters aren't working: too small to be readable, and I'm getting a missing glyph. Let's just use E and W.

<img width="68" alt="screen shot 2017-11-18 at 21 03 40" src="https://user-images.githubusercontent.com/2543659/32984313-5ce04f28-cca4-11e7-8f54-4e3992bcb256.png">

Reply:
As I stated 'configuration of the message still needs to be implemented'.
I will implement it, of course.

Reply:
okidoki, just putting it on the checklist here 👍

Reply:
OK. Implemented it. It's `W` for warnings and `E` for errors now.